mozregression/fetch_configs.py

```diff
@@ -222,11 +222,15 @@ class InboundConfigMixin(object):
     Define the inbound-related required configuration.
     """
     inbound_branch = 'mozilla-inbound'
+    branch_path = 'integration'
```

Comment:
Well, this is used just above, https://github.com/mozilla/mozregression/pull/319/files#diff-9fb3f9f996418622faaeaf680fa89a1dR84. This is the default value for https://github.com/mozilla/mozregression/pull/319/files#diff-4ac0cc6fa4a0d1bd5c2520a4969c04c4R232.